students/Don_LaVallee/lesson1/break_me.py

```diff
@@ -0,0 +1,70 @@
+   # Python Concrete Exceptions error examples
+
+import linecache
+import sys
+import traceback
+
+def PrintException():
+    print("Starting to show exceptions . . ."+"\n")
+    control = "0"
+    traceback.print_exc(file=sys.stdout)
+    print("\n")
+def Name_Error(): #1 menu item
+    print("\n"+"You selected Name_Error."+"\n")
+    print("A Name_Error exception in the break_me program is about to occur!")
+    try:
+        print("\n"+"This is a Name_Error exception example ... "  + nothing)
+    except:
+        PrintException()
+        control = "0"
+    finally:
+        print("Name_Error function is complete!"+"\n")
+def Type_Error(): #2 menu item
+    print("\n"+"You selected Type_Error."+"\n")
+    print("A Type_Error exception in the break_me program is about to occur!"+"\n")
+    i = 0
+    try:
+        print("oops" + i * 2)
+    except:
+        PrintException()
+        control = "0"
+    finally:
+        print("Type_Error function is complete!"+"\n")
+def Syntax_Error(): #3 menu item
+    print("\n"+"You selected Syntax_Error."+"\n")
+    try:
+        print("This code would not run if there was a Syntax_Error!"+"\n")
+        print("Example: whille x%2 == 0:")
+    except:
+        PrintException()
+        control = "0"
+    finally:
+            print("Syntax_Error function is complete!"+"\n")
+def Attribute_Exception_Thrower(): #4 menu item
```

Comment:
function names should be snake_case style per pep8 -- `attribute_exception_thrower`